CONTRIBUTING.md

```diff
@@ -1,7 +1,6 @@
 # Contributing
 
 The dance tool and tests run on the host; macOS, Linux, and Windows are expected to work.
-Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker; Docker Compose configuration is provided for convenience but not required.
```

Comment:
> Docker Compose configuration is provided for convenience but not required.
It's required now to run tests/mongo-tools

Reply:
Yeah, let's mention it, though it's still temporary and should be replaced in another PR